### ISSUE TITLE
navbar-right padding broken when using navbar-fixed-top with 3.3.6 

### ISSUE THREAD

**Issue by rifton007** [twbs/bootstrap#18362]
Hi all and thank you for your help.

The navbar right padding is broken when using navbar-fixed-top with Bootstrap 3.3.6.

navbar-static-top and navbar-fixed-bottom are probably impacted too.
Live example: http://plnkr.co/edit/WGhHw5?p=preview

Reference #11783

**Comment by twbs-lmvtfy** [comment]
Hi @rifton007!

You appear to have posted a live example (http://run.plnkr.co/plunks/zn1Pc0/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 28, column 9 thru column 73: Attribute ```` dropdown ```` not allowed on element ```` a ```` at this point.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by rifton007** [comment]
Now, the code is updated and is valid HTML5

**Comment by cvrebert** [comment]
Reposting to trigger fresh linting: http://plnkr.co/edit/WGhHw5?p=preview

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://run.plnkr.co/plunks/WGhHw5/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness
* line 12, column 3: [W012](https://github.com/twbs/bootlint/wiki/W012): `.navbar`'s first child element should always be either `.container` or `.container-fluid`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
@rifton007 Please revise the example again.

**Comment by cvrebert** [comment]
@mdo We've declared bankruptcy on the v3 navbar, correct? (X-Ref #16694)

**Comment by rifton007** [comment]
Yes, is correct.
http://plnkr.co/edit/XR178Y?p=streamer&s=600DUmzOZytyrhiJ

**Comment by twbs-lmvtfy** [comment]
Hi @rifton007!

You appear to have posted a live example (http://run.plnkr.co/plunks/XR178Y/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by mdo** [comment]
> We've declared bankruptcy on the v3 navbar, correct?

Yup!

**Comment by cvrebert** [comment]
Closing as WontFix.

**Comment by smit1335** [comment]
> We've declared bankruptcy on the v3 navbar, correct?

What does this mean?

**Comment by robgha01** [comment]
Congratulation everyone who use this will have to fix it themself by using hacks im one of them.... 
for other people coming here and getting disapointed look at this post for help 
http://stackoverflow.com/questions/20429229/bootstrap-3-navigation-bar-right-margin-is-not-supposed-to-be-15px

**Comment by freddyrayes** [comment]
I ran into this problem and had to override the CSS as posted in the stackoverflow link by robgha01.. Not the best solution but it works..
